lib/facebook/messenger/subscriptions.rb

```diff
@@ -8,30 +8,37 @@ module Subscriptions
 
       base_uri 'https://graph.facebook.com/v2.6/me'
 
-      def self.subscribe
-        response = post '/subscribed_apps', format: :json, query: {
-          access_token: Facebook::Messenger.config.access_token
-        }
+      module_function
+
+      def subscribe
+        response = post '/subscribed_apps'
 
         raise_errors(response)
 
         true
       end
 
-      def self.unsubscribe
-        response = delete '/subscribed_apps', format: :json, query: {
-          access_token: Facebook::Messenger.config.access_token
-        }
+      def unsubscribe
+        response = delete '/subscribed_apps'
 
         raise_errors(response)
 
         true
       end
 
-      def self.raise_errors(response)
+      def raise_errors(response)
         raise Error, response['error']['message'] if response.key? 'error'
       end
 
+      def default_options
+        super.merge(
+          format: :json,
+          query: {
+            access_token: Facebook::Messenger.config.access_token
+          }
+        )
+      end
```

Comment:
The tricky thing about `HTTParty.default_options` is that it's evaluated once at compile time and cached forever. Our access tokens don't change, though, so I guess it works in this case (but a good thing to be aware of).